espaloma/data/dataset.py

```diff
@@ -340,14 +340,14 @@ def save(self, path):
             graph.save(path + "/" + str(idx))
 
     @classmethod
-    def load(cls, path):
+    def load(cls, path, improper_def=None):
```

Comment:
same as in `esp.graph`. looks like this is changing too much of the mainline code